jetty-websocket/websocket-api/src/main/java/org/eclipse/jetty/websocket/api/FrameCallback.java

```diff
@@ -0,0 +1,53 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2017 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.websocket.api;
+
+public interface FrameCallback
+{

```

Comment:
Explain again why the normal `org.eclipse.jetty.util.Callback` cannot be used?   The implementation is using Jetty classes for IO etc. so it is dependent on `o.e.j.util`.  Or is this something that is exposed to the webapp and thus needs classloader isolation?   But then we may as well give them util as some other jar?     So if this is necessary, it at least needs javadoc saying why.